src/components/directory/DirectoryPage.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+// import VisibilityOffIcon from '@material-ui/icons/VisibilityOff';
+import {
+  Tab,
+  Tabs,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Paper
+} from '@material-ui/core';
+
+const styles = theme => ({
+  title: {
+    fontWeight: 'normal',
+    marginLeft: '30px',
+  },
+  flex: {
+    display: 'flex',
+    [theme.breakpoints.down('sm')]: {
+      display: 'block',
+    },
+  },
+  header: {
+    justifyContent: 'space-between',
+  },
+  directory: {
+    width: '100%',
+    overflowX: 'auto',
+  },
+  table: {
+    minWidth: 650,
+  },
+});
+
+class DirectoryPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: 0
+    };
+  }
+
+  render() {
+    const { classes } = this.props;
+
+    function handleChange(e, value) {
+      this.setState({ value: value });
+    }
+
+    function createData(name, job, username, password) {
+      return { name, job, username, password };
+    }
+
+    function Directory(props) {
+      const { label, rows } = props
+      return <Paper className={classes.directory}>
+        <Table className={classes.table} aria-label="simple table">
+          <TableHead>
+            <TableRow>
+              <TableCell>{label} Name</TableCell>
+              <TableCell>{label === 'Candidate' ? 'University Name' : 'Job Title'}</TableCell>
+              <TableCell>User Name</TableCell>
+              <TableCell>Password</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {rows.map(row => (
+              <TableRow key={row.name}>
+                <TableCell component="th" scope="row">
+                  {row.name}
+                </TableCell>
+                <TableCell>{row.job}</TableCell>
+                <TableCell>{row.username}</TableCell>
+                <TableCell>&bull;&bull;&bull;&bull;&bull;&bull;&bull;&bull;</TableCell>
```

Comment:
Is it possible to use something else to replace the bullet points?

Reply:
This is supposed to be a filler for now. I'm thinking of replacing it with a real password, but I wasn't for how to make them visible/masked upon toggle. What do you think?

Reply:
Sounds good to me!!